docs/content/integrations/airflow/from-airflow-to-dagster.mdx

```diff
@@ -0,0 +1,208 @@
+
+A [Job](/concepts/ops-jobs-graphs/jobs) is made up of a [Graph](/concepts/ops-jobs-graphs/graphs) of [Ops](/concepts/ops-jobs-graphs/ops). If you've used the Airflow taskflow api this will feel familiar, With ops you will be focused on writing a graph with python functions as the nodes with the data dependencies between those as edges.
+
+### Step 1: Defining the Ops
+
+After rewriting the operators of the tutorial Airflow Dag as Ops we're left with:
+
+```python file=/integrations/airflow/tutorial_rewrite_dagster.py startafter=start_ops endbefore=end_ops
+@op
+def print_date(context) -> datetime:
+    ds = datetime.now()
+    context.log.info(ds)
+    return ds
+
+
+@op(retry_policy=RetryPolicy(max_retries=3), ins={"start": In(Nothing)})
+def sleep():
+    time.sleep(5)
+
+
+@op
+def templated(context, ds: datetime):
+    for _i in range(5):
+        context.log.info(ds)
+        context.log.info(ds - timedelta(days=7))
+```
+
+this yeilds us the following graph of ops
+
+<center>
+  <Image
+    alt="Screenshot of the dagster UI, showing the newly created graph of tutorial Ops"
+    src="/images/integrations/airflow/airflow_tutorial_rewrite_ops.png"
+    width={2200}
+    height={1300}
+  />
+</center>
+
+Since Dagster is much more data oriented than Airflow you'll notice that we used params to define the execution order, not an arbitrary linking of tasks
+
+##### Op Level Retries
+
+In the tutorial Dag the sleep task allowed for 3 retries, to configure this in dagster you can use [Op level retry policies](/concepts/ops-jobs-graphs/op-retries)
+
+### Step 2: Defining the Job
+
+Now we'll define a Job, which will capture the execution order and data dependencies between our Ops.
+
+```python file=/integrations/airflow/tutorial_rewrite_dagster.py startafter=start_job endbefore=end_job
+@job(tags={"dagster/max_retries": 1, "dag_name": "example"})
+def tutorial_job():
+    ds = print_date()
+    sleep(ds)
+    templated(ds)
+```
+
+##### Job Level Retries
+
+Job level retries are managed by the [run launcher](/deployment/run-retries) you will need to enable support for them in your dagster.yaml, once you do you can define the retry count on the job.
+
+### Step 3: Defining the schedule
+
+Finally we'll add a schedule
+
+```python file=/integrations/airflow/tutorial_rewrite_dagster.py startafter=start_schedule endbefore=end_schedule
+schedule = ScheduleDefinition(job=tutorial_job, cron_schedule="@daily")
+```
+
+### Step 4: Running locally
+
+In order to run our newly defined Dagster Job we'll need to define a Dagster [Repository](/concepts/repositories-workspaces/repositories).
+
+```python file=/integrations/airflow/tutorial_rewrite_dagster.py startafter=start_repo endbefore=end_repo
+@repository
+def rewrite_repo():
+    return [tutorial_job, schedule]
+```
+
+We can now load this file with dagit, the interactive web-ui for dagster
+
+```bash
+dagit -f <your_dagster_file>.py
+```
+
+for reference here's what the file should look like
```

Comment:
This should be a separate section "## Completed Code Example", similar to what we have in Snowflake: https://docs.dagster.io/integrations/snowflake